soh/soh/resource/type/Skeleton.cpp

```diff
@@ -77,4 +81,45 @@ void SkeletonPatcher::UpdateSkeletons() {
             skel.skelAnime->skeleton = newSkel->skeletonData.skeletonHeader.segment;
     }
 }
+
+void SkeletonPatcher::UpdateTunicSkeletons() {
+    bool isHD = CVarGetInteger("gAltAssets", 0);
+    for (auto skel : skeletons) {
+        if (skel.actor != nullptr) // is there actually an actor to check?
+        {
+            if (skel.actor->id == 0) // is this Link?
+            {
+                Skeleton* newSkel;
+                if (LINK_IS_ADULT && CUR_EQUIP_VALUE(EQUIP_TUNIC) - 1 == PLAYER_TUNIC_KOKIRI &&
+                    ResourceGetIsCustomByName(gLinkKokiriSkel)) {
+                    newSkel = (Skeleton*)LUS::Context::GetInstance()
+                                  ->GetResourceManager()
+                                  ->LoadResource("__OTR__objects/object_link_kmdl/gLinkKokiriSkel", true)
```

Comment:
You can use your #defines here instead of string literals. Looks like they aren't used otherwise,

Reply:
The defines are used to check if the resource is custom, but now that you mention that it can just be used for loading it as well, I feel a bit dumb

Reply:
Double checking this, using the defines to load doesn't work, and crashes the game instead.

Reply:
Updated.